3648/3/db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+NAME = 0
+LOCATION = 1
+DESCRIPTION = 2
+RATING = 3
+
+CSV.foreach(File.realpath('db/data/places.csv')) do |row|
+  Place.create(
```

Comment:
place name should be uniq
I ran db:seed 3 times - got 9 places.
Make it so that when I run seeds multiple times - It checks if place exists and if so - it should update the place not create a new one.

Reply:
done